earthly/rust/Earthfile

```diff
@@ -84,16 +80,13 @@ rust-base:
 rust-base-all-hosts:
     BUILD --platform=linux/amd64 --platform=linux/arm64 +rust-base
 
-# Common Rust setup
+# Common Rust setup.
 # Parameters:
 #  * toolchain : The `rust-toolchain` toml file.
 SETUP:
     COMMAND
-    ARG toolchain=./rust-toolchain.toml
 
-    # Poetry Installation directory.
-    # Gives poetry and our poetry project a reliable location.
-    WORKDIR /build
+    ARG toolchain=./rust-toolchain.toml
```

Comment:
I think its better for UDC arguments to appear after the COMMAND so they are clear when looking at it.
I shouldn't have to read the whole UDC to find its arguments.